teuthology/orchestra/run.py

```diff
@@ -240,24 +240,21 @@ def __eq__(self, value):
         return self.value == value
 
 
-def quote(args):
+def convert_args_list_to_str(args):
```

Comment:
I don't think this name is much better than, just 'quote', I had no difficulties to understand what is it, though, shorter name can probably suits better: args2str or args_to_str/args_as_str, or `quoted_args`. Or if we want to be even more certain about function (sarcasm) we need to rename to **quote_args_if_required_and_return_as_space_joined_str** :-D.

Reply:
i think `quote()` is an internal method as the previous doc string explains. its only consumer in production resides in this very same file. the other 4 consumers are tests. what's the motivation to improve it?

Reply:
> i think quote() is an internal method as the previous doc string explains. its only consumer in production resides in this very same file. the other 4 consumers are tests. what's the motivation to improve it?

It has been used in vstart_runner.py as well - https://github.com/ceph/ceph/blob/master/qa/tasks/vstart_runner.py#L441.

Motivation: currently, run.quote() is an teuthology specific extension of pipes.quote() (since it deals with Raw instances and list args as well) and there's no hint of it in the name or docstring. If that wasn't the case, it would be easier to know and therefore use it.